src/libs/actions/IOU.js

```diff
@@ -117,6 +117,127 @@ Onyx.connect({
     },
 });
 
+/**
+ * Initialize money request info
+ * @param {String} reportID to attach the transaction to
+ * @param {String} [iouRequestType] one of manual/scan/distance
+ */
+function startMoneyRequest_temporaryForRefactor(reportID, iouRequestType = CONST.IOU.REQUEST_TYPE.MANUAL) {
+    // Generate a brand new transactionID
+    const newTransactionID = CONST.IOU.OPTIMISTIC_TRANSACTION_ID;
+    const created = currentDate || format(new Date(), 'YYYY-MM-DD');
```

Comment:
```suggestion
    const created = currentDate || format(new Date(), 'yyyy-MM-dd');
```
This was causing a crash. Apparently `date-fns` uses unicode tokens, so `YYYY` represents the local week-numbering year instead, more info [here](https://github.com/date-fns/date-fns/blob/main/docs/unicodeTokens.md).